internal/utils/parse_test.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseInt32(t *testing.T) {
+	for _, tc := range []struct {
+		description string
+		input       string
+		result      int32
+		err         string
+	}{
+		{
+			description: "regular",
+			input:       "10",
+			result:      10,
+		},
+		{
+			description: "overflow",
+			input:       strconv.Itoa(math.MaxInt64),
+			err:         "strconv.ParseInt: parsing \"%s\": value out of range",
```

Comment:
Instad of comparing strings, can we unwrap the received error and compare with defined error values.
e.g.: https://golang.org/pkg/strconv/#pkg-variables